src/app/pages/ArticlePage/ContinueReadingButton/index.tsx

```diff
@@ -0,0 +1,121 @@
+/** @jsx jsx */
+import React, { use, useEffect } from 'react';
+import { jsx } from '@emotion/react';
+import Text from '#app/components/Text';
+import { EventTrackingMetadata } from '#app/models/types/eventTracking';
+import useClickTrackerHandler from '#app/hooks/useClickTrackerHandler';
+import useViewTracker from '#app/hooks/useViewTracker';
+import { ServiceContext } from '#app/contexts/ServiceContext';
+import styles from './index.styles';
+
+type Props = {
+  showAllContent: boolean;
+  setShowAllContent: () => void;
+  variation: 'A' | 'B' | null;
+  eventTrackingData?: EventTrackingMetadata;
```

Comment:
Should this be optional given its an experiment? I figured we'd always want to have it to validate results

Reply:
We can delete this type property. EventTrackingMetadata can be defined directly in this component, so it doesn't need to be passed in as a prop.

Reply:
removed